src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const myCompareFunction = (a, b) => {
+    const aString = String(a);
+    const bString = String(b);
+
+    if (aString > bString) {
+      return 1;
+    } else if (aString === bString) {
+      return 0;
+    } else {
+      return -1;
+    }
+  };
+
+  [].__proto__.sort2 = function(compareFunction = myCompareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let y = 1; y < this.length; y++) {
```

Comment:
this is wrong implementation of how method sort should work, read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)